src/Language/PureScript/Parser/Declarations.hs

```diff
@@ -281,6 +285,13 @@ parseModulesFromFiles toFilePath input = do
   inParallel :: [Either P.ParseError (k, [Module])] -> [Either P.ParseError (k, [Module])]
   inParallel = withStrategy (parList rseq)
 
+  updatePositions :: [PositionedToken] -> [PositionedToken]
```

Comment:
Why not put this function in the lexer module, or even fuse it into the lexer function?